src/cryptography/sign.js

```diff
@@ -24,8 +26,25 @@ const signatureHeader = createHeader('SIGNATURE');
 const secondSignatureHeader = createHeader('SECOND SIGNATURE');
 const signatureFooter = createHeader('END LISK SIGNED MESSAGE');
 
-export const signMessageWithPassphrase = (message, passphrase) => {
+const SIGNED_MESSAGE_PREFIX = Buffer.from('Lisk Signed Message:\n', 'utf8');
```

Comment:
If `'Lisk Signed Message:\n'` is an constant, shouldn't we put this in constant file?
Also, do we need line break?

Reply:
All implementation for all other coins use the line break, so I think Lisk should use the same way too